### PR TITLE
RothC config working with all the parameters 

### LINKED ISSUE #20: Send RothC config to endpoint
We can now edit the RothC config which is fantastic, but it's not yet making it out of the client to the `rothc` endpoint on our backend. 

Therefore we need to replicate the logic from the `send_pointConfig` action for RothC. Example at: https://github.com/moja-global/FLINT-UI/blob/master/flint.ui/src/store/index.js#L391

### PR DESCRIPTION
**Pull request by waridrox**
Fixes #20 

All the params for the RothC model are being saved as states until the final run button is invoked.

https://user-images.githubusercontent.com/58583793/127323344-0660f319-762e-4943-b81b-244a22393d15.mov

### CODE REVIEW

File: flint.ui/src/views/flint/configurations/rothc/RothcSoilcover.vue [hunk 1]
```diff
@@ -61,30 +76,54 @@
...
       set(newValue) {
-        // console.log(newValue)
         let tempval = newValue;
-        this.$store.commit('setNewConfig', tempval)
+        this.$store.commit('setNewConfig_soilCover_data_orig', tempval)
       }
-    }
+    },
+    
+    newconfig_soilcover_data_firstmonthcopy: {
```
Comment (aornugent) [inline flint.ui/src/views/flint/configurations/rothc/RothcSoilcover.vue:94]: @waridrox - have you been able to verify whether any of these variables, other than `*_data_month_avg` are necessary to run?

If not, please file an issue and request they be removed to check whether the endpoint still works.

File: flint.ui/src/views/flint/configurations/rothc/RothcSoilcover.vue [hunk 2]
```diff
@@ -103,14 +142,14 @@ export default {
...
+         console.log(this.$store.state.RothC_config.Variables[12].soilCover.transform.data_orig);
       console.log("Sss");
     },
-    foobar (event) {
+    // foobar (event) {
     
-         console.log(event.target.value);
-        console.log(this.$store.state.Variables[8].rainfall.transform.data_orig);
+    //      console.log(event.target.value);
+    //     console.log(this.$store.state.RothC_config.Variables[12].soilCover.transform.data_orig);
```
Comment (aornugent) [inline flint.ui/src/views/flint/configurations/rothc/RothcSoilcover.vue:152]: Is this dead code?

**Comment by aornugent** [comment]
Nice work @waridrox! These look great. I think this address #20. 

When submitting PRs, can you please write a short update on what features are being added? Just a couple of bullets is all we need, but it makes it much easier to manage the assess the PR and track what happened in our git history.

File: flint.ui/src/views/flint/configurations/rothc/RothcSoilcover.vue (hunk 2 above)
Comment (waridrox) [reply flint.ui/src/views/flint/configurations/rothc/RothcSoilcover.vue:152]: Hi @aornugent, yes I think the code needs more cleaning...

**Pull request merged**